src/main/kotlin/com/dansplugins/fiefs/fief/Fief.kt

```diff
@@ -0,0 +1,38 @@
+package com.dansplugins.fiefs.fief
+
+import com.dansplugins.factionsystem.faction.MfFactionId
+import com.dansplugins.factionsystem.player.MfPlayerId
+import java.util.UUID
+
+data class Fief(private val name: String, private val ownerMfPlayerId: MfPlayerId, private val mfFactionid: MfFactionId) {
+    private var id = MfFiefId.generate()
+    private var members: MutableList<MfPlayerId> = mutableListOf()
```

Comment:
Unless you're replacing the entire list, this can be a val.
Additionally, if you use a List instead of a MutableList, this class can be immutable.
id and members could probably be moved into the constructor and given a default value.